.travis.yml

```diff
@@ -1,45 +0,0 @@
-language: bash
-
-env:
-  global:
-    - REPO="${TRAVIS_REPO_SLUG#*/}"
-    - OWNER=plus3it
-    - RELEASE_BRANCH=master
-    - RELEASE_VERSION=$(grep -E '^current_version' $TRAVIS_BUILD_DIR/.bumpversion.cfg | sed 's/^.*= //')
-
-if: branch = master OR type = pull_request OR tag is present
-
-jobs:
-  include:
-    - stage: lint
-      name: Project Syntax Verification
-      script: make && make docker/run target=lint
-    - stage: lint
-      name: "CFN Version Validation"
-      install:
-        - pip install -r _requirements/cfn-version-check.txt
-        - yq --version
-      language: python
-      script: make cfn/version VERSION=$RELEASE_VERSION
```

Comment:
This project has a "custom" job that will require a new workflow, running `make cfn/version VERSION=$RELEASE_VERSION`... It's unique to this project, so I think just add a new workflow file that is triggered on both pull_request and workflow_call to run that command. Then update the release.yaml workflow to call that workflow as a reusable job, with `needs` on the release job. That should chain everything together well enough.